WORKSPACE

```diff
@@ -59,6 +59,32 @@ http_archive(
     ],
 )
 
+# Initialize hermetic Python
+load("@xla//third_party/py:python_init_rules.bzl", "python_init_rules")
+
+python_init_rules()
+
+load("@xla//third_party/py:python_init_repositories.bzl", "python_init_repositories")
+
+python_init_repositories(
+    requirements = {
+        "3.10": "//:requirements_lock_3_10.txt",
```

Comment:
We need 3.8 - 3.11 support, as we release whl for all python versions

Reply:
Ok, I'll add (in  new brahcn-based PR) 3.8 as well.

Reply:
Sounds good, thanks!